### DISCUSSION TITLE
皮肤配置在ubuntu系统下有作用吗，我的不起作用

### DISCUSSION THREAD

**Discussion by yuexl** [KyleBing/rime-wubi86-jidian#101] [Q&A] [answered]
ubunut20.04  配置在.config/ibus/rime/build下
![image](https://github.com/KyleBing/rime-wubi86-jidian/assets/26302906/7abb5191-d4d4-49c1-935c-12a35282acb2)

**Comment by KyleBing** [comment, marked as answer]
Ubuntu 没有定义皮肤的功能。

**Reply by yuexl** [reply]
感谢回复。配置rime没用，不过通过ibus配置了皮肤可以使用。
